setup.py

```diff
@@ -573,6 +576,21 @@ def get_ext_modules() -> List[Extension]:
                 "executorch/examples/models/llama2/custom_ops",
             )
         )
+        ext_modules.append(
+            # Install the prebuilt library for quantized ops required by custom ops.
+            BuiltFile(
+                "kernels/quantized/libquantized_ops_aot_lib.*",
+                # With "executorch/kernels/quantized" it produces a file instead of putting to that dir
+                "executorch/examples/models/llama2/custom_ops",
+            )
+        )
+        ext_modules.append(
+            # Install the prebuilt library required by quantized ops
+            BuiltFile(
+                "libexecutorch_no_prim_ops_shared.*",
+                "executorch",
```

Comment:
Something is wrong here. `portable_lib` already contains all of the code that's provided by `libexecutorch_no_prim_ops_shared`. If both of these .so files are loaded at the same time we'll get some kind of symbol conflict. And more confusingly, there will be two copies of the kernel and backend registries. If a kernel is registered into one, it won't be visible in the other one.